support-frontend/assets/pages/digital-subscriber-checkout/digitalSubscriptionLandingPage.tsx

```diff
@@ -139,7 +139,7 @@ export function SupporterPlusLandingPage({
 
 	return (
 		<PageScaffold
-			id="supporter-plus-landing"
+			id="digital-subscription-checkout-page"
```

Comment:
Is this 
```suggestion
			id="digital-subscription-landing-page"
```

?

Reply:
Personally I thought it is a checkout not a landing page but then will need to rename the whole file if take this further so have lined it up to the suggestion, added.